lib/tools/system-calls.js

```diff
@@ -434,6 +434,11 @@ systemCallMethods.launchAVD = async function (avdName, avdArgs, language, countr
   proc.on('output', (stdout, stderr) => {
     log.info(`[AVD OUTPUT] ${stdout || stderr}`);
   });
+  proc.on('exit', (code, signal) => {
+    if (code !== 0) {
+      log.errorAndThrow(`[AVD OUTPUT] avd exit with code ${code}, signal ${signal}`);
```

Comment:
Not sure about the `[AVD OUTPUT]` part here, since this is not technically AVD output.